be/src/env/env.h

```diff
@@ -26,6 +26,15 @@ struct WritableFileOptions;
 struct RandomAccessFileOptions;
 struct RandomRWFileOptions;
 
+struct SpaceInfo {
+    // Total size of the filesystem, in bytes
+    int64_t capacity;
```

Comment:
why not give the variable an initial value. such as 0

Reply:
updated